### ISSUE TITLE
admin dependency cause problem with multi-host slaves

### ISSUE THREAD

**Issue by TA2k** [ioBroker/create-adapter#131]
Hi
the  dependency
```
"dependencies": [
            {
                "admin": ">=3.0.0"
            },
```
cause problem on multihost systems slaves without admin. What is the reason for the dependency?

**Comment by AlCalzone** [comment]
The template only includes a `index_m.html` which requires admin v3 to work.
The check itself is bugged and has to be fixed in js-controller: https://github.com/ioBroker/ioBroker.js-controller/issues/325

**Comment by AlCalzone** [comment]
Should be fixed by #199
